nrf_security/configs/psa_crypto_config.h.template

```diff
@@ -385,4 +385,9 @@
 #cmakedefine MBEDTLS_MPI_WINDOW_SIZE       @MBEDTLS_MPI_WINDOW_SIZE@ /**< Maximum window size used. */
 #cmakedefine MBEDTLS_MPI_MAX_SIZE          @MBEDTLS_MPI_MAX_SIZE@ /**< Maximum number of bytes for usable MPIs. */
 
+/* TF-M has dependency on mbedtls_hkdf function in order to avoid a recursion problem.
+ * See tfm_builtin_key_loader.c for explanation.
+ */
+#cmakedefine MBEDTLS_HKDF_C                @MBEDTLS_HKDF_C@
```

Comment:
Nit: We probably could standardize on the naked #defines for legacy configurations. It is wrong that I've started to use the #define SOMTHING 1 for these.

You can end up with "already defined"-issues...